Bottom: dabf474f7c0e6f3bdac85b46f909266d8d70e8a6
Top:    5ec6d3ef1756ed0ece94a823e935dc021fe2be06
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-01-22 17:33:54 +0000

Refresh of other-drawing-api-changes

---

diff --git a/src/draw.c b/src/draw.c
index 9fe8c77..fc47717 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -124,7 +124,7 @@ set_color_for_via (DrawAPI *dapi, PinType *via)
 }
 
 static void
-set_color_for_pad (DrawAPI *dapi, Layer *layer, PadType *pad)
+set_color_for_pad (DrawAPI *dapi, LayerType *layer, PadType *pad)
 {
   char *color = PCB->PinColor;
 
diff --git a/src/outline_draw.c b/src/outline_draw.c
index 3ec45f8..0341b04 100644
--- a/src/outline_draw.c
+++ b/src/outline_draw.c
@@ -122,10 +122,10 @@ outline_draw_pcb_arc (DrawAPI *dapi, LayerType *layer, ArcType *arc)
 }
 
 static void
-outline_draw_pcb_text (DrawAPI *dapi, LayerType *layer, TextType *text, coord min_width)
+outline_draw_pcb_text (DrawAPI *dapi, LayerType *layer, TextType *text, Coord min_width)
 {
   BoxType *box = &text->BoundingBox;
-  gui->draw_rect (Crosshair.GC, box->X1, box->Y1, box->X2, box->Y2);
+  dapi->gapi->draw_rect (dapi->gc, box->X1, box->Y1, box->X2, box->Y2);
 }
 
 /* ---------------------------------------------------------------------------
